packages/react/src/LabelGroup/LabelGroup.tsx

```diff
@@ -136,18 +132,16 @@ const OverlayToggle: React.FC<
       )}
       focusZoneSettings={{disabled: true}}
     >
-      <Box alignItems="flex-start" display="flex" width={overlayWidth} padding={`${overlayPaddingPx}px`}>
-        <Box display="flex" flexWrap="wrap" sx={{gap: 1}}>
-          {children}
-        </Box>
+      <div className={classes.Div_0} style={{width: overlayWidth, padding: `${overlayPaddingPx}px`}}>
+        <div className={classes.Div_1}>{children}</div>
```

Comment:
```suggestion
      <Stack align="start" style={{width: overlayWidth, padding: `${overlayPaddingPx}px`}}>
        <StackItem><Stack wrap="wrap" gap="condensed">{children}</Stack></StackItem>
```

I guess `condensed` is 8px, though 🤷